node/src/chain_spec/cs_eden.rs

```diff
@@ -214,6 +197,66 @@ pub fn local_config(id: ParaId) -> ChainSpec {
     )
 }
 
+fn valid_config_genesis(id: ParaId) -> GenesisConfig {
+    let valid_01_seed =
+        "castle fiscal easy grief horse include alley next youth deny engine response";
+    let valid_02_seed =
+        "giggle august whisper someone zoo trend evolve response bird board gaze pull";
+    let valid_03_seed =
+        "ramp clever visit pottery thunder fatal crawl already syrup gaze pride cabin";
+    let valid_04_seed =
+        "budget shock sudden axis predict soldier elder valve fame theme deliver slow";
+    let valid_05_seed = "party history head few eye bird exit axis flavor coconut kick robust";
+
```

Comment:
we probably do not want mnemonics to be publicly available here :)

Reply:
I agree, I think these values should be provided as environment secrets and injected into the build if necessary.

Reply:
HaHa, thought of having common validation keys  for team, allocated only for internal validation, so that like alice & bob in polkadot js , it's available once injected. If it doesn't look good. will kick it out.

Reply:
You should be able to use the [`env!()` macro](https://doc.rust-lang.org/std/macro.env.html) in cases where panicking on an unprovided value is acceptable.